specs/merge/client-settings.md

```diff
@@ -1,26 +1,34 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
-
-- [The Merge -- Client Settings](#the-merge----client-settings)
-    - [Override terminal total difficulty](#override-terminal-total-difficulty)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
-
 # The Merge -- Client Settings
 
 **Notice**: This document is a work-in-progress for researchers and implementers.
 
 This document specifies configurable settings that clients must implement for the Merge.
 
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Override terminal total difficulty](#override-terminal-total-difficulty)
+- [Override terminal block hash](#override-terminal-block-hash)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
 ### Override terminal total difficulty
 
-To coordinate manual overrides to [`terminal_total_difficulty`](fork-choice.md#transitionstore), clients
+To coordinate manual overrides to [`terminal_total_difficulty`](./fork-choice.md#transitionstore), clients
 must provide `--terminal-total-difficulty-override` as a configurable setting.
 
 If `TransitionStore` has already [been initialized](./fork.md#initializing-transition-store), this alters the previously initialized value of
-`TransitionStore.terminal_total_difficulty`, otherwise this setting initializes `TransitionStore` with the specified, bypassing `compute_terminal_total_difficulty` and the use of an `anchor_pow_block`.
-`terminal_total_difficulty`.
+`TransitionStore.terminal_total_difficulty`, otherwise this setting initializes `TransitionStore` with the specified `terminal_total_difficulty`, bypassing `compute_terminal_total_difficulty` and the use of an `anchor_pow_block`.
```

Comment:
We should also set `terminal_block_hash` field to the default `Hash32()` value in this case.